DB/src/main/java/io/deephaven/db/v2/SourceTable.java

```diff
@@ -263,8 +263,6 @@ protected void instrumentedRefresh() {
                     //noinspection unchecked
                     return new EmptyToNullStringRegionedColumnSource(source, () ->
```

Comment:
You should just delete this method (`com.illumon.iris.db.v2.SourceTable#maybeReplaceColumnSources`), remove its single use, delete the field `com.illumon.iris.db.v2.SourceTable#MAP_EMPTY_STRINGS_TO_NULL`, and delete the class `com.illumon.iris.db.v2.sources.EmptyToNullStringRegionedColumnSource`.